SWEN766BetterMaps/app/src/main/java/com/example/swen766_bettermaps/MapUtils.java

```diff
@@ -0,0 +1,103 @@
+package com.example.swen766_bettermaps;
+
+import android.graphics.Color;
+import android.os.AsyncTask;
+import android.util.Log;
+
+import com.google.android.gms.maps.GoogleMap;
+import com.google.android.gms.maps.model.LatLng;
+import com.google.android.gms.maps.model.PolylineOptions;
+import com.google.maps.android.PolyUtil;
+
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.Iterator;
+import java.util.List;
+
+public class MapUtils {
+
+    public static void drawRoute(GoogleMap map, Route route, String apiKey) {
```

Comment:
No need to take in the apiKey as a parameter, since we have it saved in the secrets file